infinispan-embedded/src/test/java/org/hibernate/ogm/datastore/infinispan/test/storedprocedures/NotReallyAProcedure.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.infinispan.test.storedprocedures;
+
+/**
+ * @author The Viet Nguyen &amp;ntviet18@gmail.com&amp;
+ */
+public class NotReallyAProcedure implements Runnable {
```

Comment:
`NoOpProcedure` would be a better name

Reply:
Should we support Runnable as a StoredProcedure? Currently, we don't support it. But it will make a valid procedure which return Void.

Reply:
I thought that the `NotReally` part referred to the fact that the StoreProcedure doesn't do anything.

No need to support Runnable, but let's call it `InvalidStoredProcedure`. I think it's clearer that way. Also a bit of javadoc can go a long way in these cases.

Reply:
You could also throw an exception in the run method to make sure that no test can execute it